help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -216,11 +228,13 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 
 ### Unknown errors encountered while pushing events to Salesforce Marketing Cloud {#unknown-errors}
 
-When checking a dataflow run, you might encounter the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
+* When checking a dataflow run, you might encounter the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
+    ![Platform UI screenshot showing error.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/error.png)
 
-![Platform UI screenshot showing error.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/error.png)
+    * To fix this error, verify that the **[!UICONTROL Mapping ID]** that you provided in [!DNL Salesforce Marketing Cloud] for your Platform segment is valid and exists within [!DNL Salesforce Marketing Cloud].
 
-To fix this error, verify that the **[!UICONTROL Mapping ID]** that you provided in [!DNL Salesforce Marketing Cloud] for your Platform segment is valid and exists within [!DNL Salesforce Marketing Cloud].
+* When activating a segment, if you obtain an error message: `The client's IP address is unauthorized for this account. Allowlist the client's IP address...`
+    * To fix this error, contact your [!DNL Salesforce Marketing Cloud] account administrator to add Platform IP's to your [!DNL Salesforce Marketing Cloud] accounts' trusted IP ranges. Refer to the [!DNL Salesforce Marketing Cloud] [IP Addresses for Inclusion on Allowlists in Marketing Cloud](https://help.salesforce.com/s/articleView?id=sf.mc_es_ip_addresses_for_inclusion.htm&type=5) documentation if you need additional guidance.
```

Comment:
As indicated by Nipun, let's use this page for the Adobe IPs: 

`[...] to add [Experience Platform IP addresses](/help/destinations/catalog/streaming/ip-address-allow-list.md) to your [...]`